src/linker/Linker.Dataflow/Helpers.cs

```diff
@@ -0,0 +1,23 @@
+using System.Runtime.CompilerServices;
+
+using Mono.Cecil;
+
+namespace Mono.Linker.Dataflow
+{
+	static class TypeHelper
+	{
+		public static bool TypesAreEquivalent (TypeDefinition t1, TypeDefinition t2)
+		{
+			return RuntimeHelpers.Equals (t1, t2);
+		}
+
+		public static int GetHashCode (TypeDefinition t) => t.GetHashCode ();
```

Comment:
What is the value of this?

Reply:
`Object.GetHashCode`. We'll probably want to implement something better. We'll find out when we do more detailed perf measurements, but it might not matter since this only affects merging stack slots that model System.Type/RuntimeTypeHandle and I don't expect we would see many.